tests/testthat/test-sparsevctrs.R

```diff
@@ -0,0 +1,60 @@
+test_that("sparse tibble can be passed to `fit() - recipe", {
+  skip_if_not_installed("xgboost")
```

Comment:
Is there a lighter-dependency engine that accepts sparsity in parsnip? This makes me wonder if we ought to test in extratests.

Reply:
glmnet would be one and has been helpful in surfacing all kinds of subtle things

Reply:
switched to glmnet in [`84b21c0` (#245)](https://github.com/tidymodels/workflows/pull/245/commits/84b21c040af52e645cb1185d145d56810b933944)